### PR TITLE
components: Switch

### PR DESCRIPTION
**Pull request by siddharthkp**
Added a switch component

also fixed 2 unrelated bugs
1. theme in story/config
2. grid column types

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3296/commits/32a5322d0e338a7371250efbcd75757f81e151b9 failed.

File: packages/components/src/components/Switch/switch.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  title: 'components/Switch',
+  component: Switch,
+};
+
+export const Basic = () => <Switch />;
+
+export const defaultOn = () => <Switch defaultOn />;
+
+/* eslint-disable no-console */
+export const onChange = () => <Switch onChange={action('changed')} />;
```
Comment (SaraVieira) [inline packages/components/src/components/Switch/switch.stories.tsx]: This makes the console log be on the addons tab

File: packages/components/src/components/Switch/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+interface ISwitchProps {
+  on?: boolean;
+  defaultOn?: boolean;
+  onChange?: (event: any) => void;
+}
+
+export const Switch: React.FC<ISwitchProps> = ({ on, defaultOn, ...props }) => (
+  <SwitchContainer as="label">
+    <SwitchInput
```
Comment (SaraVieira) [inline packages/components/src/components/Switch/index.tsx]: This input doesn't have a label maybe we need something here?

Comment (siddharthkp) [reply packages/components/src/components/Switch/index.tsx]: The label is given by the label on the left

Screenshot 2020-01-09 at 8 38 19 AM

How do we suggest folks to attach them together with for and id, one way could be to make `id` compulsory on the switch and throw a warning if it's not given?

File: packages/components/src/components/Switch/switch.stories.tsx (hunk 1 above)
Comment (siddharthkp) [reply packages/components/src/components/Switch/switch.stories.tsx]: amazing,thanks

File: packages/components/src/components/Switch/index.tsx (hunk 2 above)
Comment (SaraVieira) [reply packages/components/src/components/Switch/index.tsx]: Sounds good!

**Pull request merged**